pkg/config/paths.go

```diff
@@ -0,0 +1,46 @@
+package config
+
+import (
+	"fmt"
+	URL "net/url"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/PuerkitoBio/purell"
+)
+
+func GetBase(base *string) string {
+	if base == nil || *base == "" {
+		baseUrl, err := os.Getwd()
+		if err != nil {
+			return ""
+		}
+		return baseUrl
+	} else {
+		return *base
+	}
+}
+
+func NormalizeUrl(url string, base ...string) (string, error) {
+	var normalUrl *URL.URL
+	var err error
+	var baseUrl string
+	if len(base) > 0 {
+		baseUrl = GetBase(&base[0])
+	} else {
+		baseUrl = GetBase(nil)
+	}
+	if strings.HasPrefix(url, ".") {
+		normalUrl, err = URL.Parse(fmt.Sprintf("file:///%s", path.Join(baseUrl, url)))
+	} else if strings.HasPrefix(url, "/") {
+		normalUrl, err = URL.Parse(fmt.Sprintf("file:///%s", url))
+	} else {
```

Comment:
Would it make sense for another case to look `http://` or `https://` and the else case using `path.Join`? When working with local files, I personally dislike having to remember to prefix with `./`. I get the "modules"/ESM approach though.

Reply:
good point. I'll invert the logic and parse it as a URL and if there's an error or the scheme is `''` then I'll treat it as a local file.